src/core/data-tests.js

```diff
@@ -105,7 +105,7 @@ function toTestURLs(tests, testSuiteURI) {
       try {
         return new URL(test, testSuiteURI).href;
       } catch {
-        pub("warn", `${l10n[lang].bad_uri}: ${test}`);
+        pub("warn", `Bad URI: ${test}`);
```

Comment:
This was a missing l10n string in all languages :tada: 
Kinda caught by TypeScript (not lint error sadly, but shows up as "any" instead of "string" and does't appear in VSCode's suggestions).